app/src/main/java/org/mozilla/focus/ScreenController.kt

```diff
@@ -92,7 +92,7 @@ object ScreenController {
             browserFragment.loadUrl(url)
         } else {
             val session = Session(url, source = source)
-            context.components.sessionManager.add(session)
+            context.components.sessionManager.add(session, true)
```

Comment:
nit: Do you think it would be more readable to use a named parameter like `selected = true` here?

Reply:
Good call, I'll make that change.